src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubMediator.cs

```diff
@@ -588,7 +578,7 @@ private void HandlePrune()
 
         private void HandleGossip()
         {
-            var node = SelectRandomNode(_nodes.Except(new[] { _cluster.SelfAddress }).ToArray());
+            var node = SelectRandomNode(_nodes.Except([_cluster.SelfAddress]).ToArray());
```

Comment:
You might be able to make this work without allocating a list - have `SelectRandomNode` take an `IEnumerable` and just apply the random value to `.Skip` instead of the indexer

Reply:
That'll also save you the trouble of fully evaluating the list

Reply:
I'll give that a try